### PR TITLE
Updates to Network Policy Rule Targets component

### LINKED ISSUE #4729: Feature Request: Add edit and detail UI for NetworkPolicies
**Detailed Description**
Add a UI to edit and viel NetworkPolicies.

**Context**
NetworkPolicies are one of the more complex Kubernetes resources. But they are also frequently used in enterprise contexts to restrict incoming and outgoing traffic of pods. A UI would make this much more accessible, especially for Kubernetes beginners.

### PR DESCRIPTION
**Pull request by richard-cox**
- Move cidr/s validation messages underneath cidr/s fields
- Fix error thrown when switching from ip block to other rule source options
- fixes #4729

### CODE REVIEW

File: edit/networking.k8s.io.networkpolicy/PolicyRuleTarget.vue [hunk 1]
```diff
@@ -225,7 +216,14 @@ export default {
           />
         </div>
       </div>
-      <div class="row mb-0">
+      <div v-if="invalidCidr" class="row">
+        <div class="col span-12">
+          <Banner color="error">
+            <t v-if="invalidCidr" k="networkpolicy.rules.ipBlock.invalidCidr" />
```
Comment (bashofmann) [inline edit/networking.k8s.io.networkpolicy/PolicyRuleTarget.vue:222]: Nit: the `v-if` condition is already checked in the `div` above

File: edit/networking.k8s.io.networkpolicy/PolicyRuleTarget.vue [hunk 2]
```diff
@@ -237,6 +235,13 @@ export default {
           />
         </div>
       </div>
+      <div v-if="invalidCidrs.length" class="row mb-10">
+        <div class="col span-12">
+          <Banner color="error">
+            <t v-if="invalidCidrs.length" k="networkpolicy.rules.ipBlock.invalidExceptionCidrs" />{{ invalidCidrs.join(', ') }}
```
Comment (bashofmann) [inline edit/networking.k8s.io.networkpolicy/PolicyRuleTarget.vue:241]: Nit: the `v-if` condition is already checked in the `div` above

**Review by bashofmann — APPROVED** [review]

File: edit/networking.k8s.io.networkpolicy/PolicyRuleTarget.vue (hunk 2 above)
Comment (richard-cox) [reply edit/networking.k8s.io.networkpolicy/PolicyRuleTarget.vue]: Good catch! I've removed the check here and on the text for `invalidCidr` as well

**Review by nwmac — APPROVED** [review]
LGTM

**Pull request merged**